### PR TITLE
CE#1794: Move output-producer bakery scripts into Enki instead of being a submodule

### PR DESCRIPTION
**Pull request by sparksam**
@philschatz Is it okay if the scripts are located `output-producer-service/bakery` ? I was able to run the tests and build few books.

### CODE REVIEW

**Comment by therealmarv** [comment]
Question (not related to this PR): What happens with the bakery scripts in output-producer-service / CORGI ?

**Review by therealmarv — APPROVED** [review]

**Review by philschatz — CHANGES_REQUESTED** [review]
I think most of the files in the bakery directory are no longer useful and could be excluded from the PR. I think pretty much everything except the python files in bakery/src/scripts/ are no longer used.

Also, do you know how to keep code coverage of the python files?

**Comment by sparksam** [comment]
@philschatz Please review.

**Review by philschatz — CHANGES_REQUESTED** [review]
I am having trouble finding hte code coverage information from the bakery scripts tests. In codecov I still only see coverage for dockerfiles and build-concourse (the shell scripts and the JS) on the [sk-ce-1794 branch](https://codecov.io/gh/openstax/enki/tree/sk-ce-1794)

Also, these files used by anything in the repo?

- bakery/src/scripts/Dockerfile
- bakery/src/scripts/*.sh and maybe most of *.json

File: test/test-step-10.bash [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+[[ $TRACE_ON ]] && set -x
+[[ $0 != "-bash" ]] && cd "$(dirname "$0")"
+
+pip install "../output-producer-service/bakery/src/scripts/.[test]"
```
Comment (philschatz) [inline test/test-step-10.bash:6]:
```suggestion
if [[ $CI = '' && $VIRTUAL_ENV = '' ]]; then
   echo "Activate a virtualenv before running this"
   exit 1
fi

pip install "../output-producer-service/bakery/src/scripts/.[test]"
```

Will this install packages into the system version of python?

**Review by philschatz — APPROVED** [review]

**Pull request merged**